### ISSUE TITLE
Switch to rewrite-java-dependencies for Maven & Gradle dependency recipes

### ISSUE THREAD

**Issue by timtebeek** [quarkusio/quarkus-updates#65]
In [openrewrite/rewrite-java-dependencies](https://github.com/openrewrite/rewrite-java-dependencies) there's recipes that abstract over Maven and Gradle to update both. That means some of the [maven specific recipes in this repository](https://github.com/search?q=repo%3Aquarkusio%2Fquarkus-updates+org.openrewrite.maven+language%3AYAML&type=code&l=YAML) can be replaced by recipes that also update Gradle projects, quite easily.

As an example, see the following pull request and suggested changes
- https://github.com/openrewrite/rewrite-logging-frameworks/pull/123/files

In short it comes down to
1. Add a dependency on `org.openrewrite.recipe:rewrite-java-dependencies`
2. Optionally remove rewrite-maven & rewrite-gradle, they're transitive through above
3. Replace `org.openrewrite.maven` with `org.openrewrite.java.dependencies`
4. Replace `ChangeDependencyGroupIdAndArtifactId` with `ChangeDependency`

Should be a quick win for folks looking to use these recipes on Gradle projects.

**Comment by gsmet** [comment]
This was included in https://github.com/quarkusio/quarkus-updates/pull/64 .
